src/main/java/ch/naviqore/gtfs/schedule/type/RouteTypeMapper.java

```diff
@@ -0,0 +1,45 @@
+package ch.naviqore.gtfs.schedule.type;
+
+public class RouteTypeMapper {
+
+    public static DefaultRouteType map(RouteType routeType) {
+        return switch (routeType) {
+            case null -> null;
```

Comment:
Why can this be null? I would not handle this case... Otherwise we are ending up passing nulls around and have to check everywhere. Better introduce a RouteType.UNKOWN if needed.

Reply:
Removed in 971b15e